math/mobius/mobius.go

```diff
@@ -0,0 +1,44 @@
+// mobius.go
+// description: Returns μ(n)
+// details:
+// For any positive integer n, define μ(n) as the sum of the primitive nth roots of unity.
+// It has values in {−1, 0, 1} depending on the factorization of n into prime factors:
+//   μ(n) = +1 if n is a square-free positive integer with an even number of prime factors.
+//   μ(n) = −1 if n is a square-free positive integer with an odd number of prime factors.
+//   μ(n) = 0 if n has a squared prime factor.
+// wikipedia: https://en.wikipedia.org/wiki/M%C3%B6bius_function
+// author: Akshay Dubey (https://github.com/itsAkshayDubey)
+// see mobius_test.go
+
+package mobius
+
+import (
+	p "github.com/TheAlgorithms/Go/math/prime"
+)
+
+// Mu is the Mobius function
+// This function returns μ(n) for given number
+func Mu(n int) int {
+
+	if n <= 1 {
+		return 1
+	}
+
+	primeFactorCount := 0
```

Comment:
If the first meaningful value for you is the default value then prefer declaration over initialisation. Though arguably its more verbose and its a matter of taste so I leave it up to you to either change it or leave it as is 😄 

```suggestion
	primeFactorCount := 0
```

Reply:
Fixed this, in this [commit](https://github.com/TheAlgorithms/Go/pull/532/commits/99228ad2465fc6fba55b08f7b1b10df9153601c1).